src/components/NewLexemeForm.vue

```diff
@@ -190,31 +190,31 @@ export default {
 </template>
 
 <style scoped lang="scss">
-@import '@wmde/wikit-tokens/variables';
-@import '@wmde/wikit-vue-components/src/styles/mixins/Typography';
+@import '@/styles/custom-variables.css';
+@import '@wikimedia/codex-design-tokens/theme-wikimedia-ui';
 
 .wbl-snl-form {
 	& > * + * {
-		margin-top: $dimension-layout-xsmall;
+		margin-top: var( --dimension-layout-xsmall );
 	}
 
 	// Box model
-	padding: $dimension-layout-small;
+	padding: var( --dimension-layout-xsmall );
```

Comment:
This is supposed to be `--dimension-layout-small`, not **x**small, according to T369510. (Assuming we want to preserve the previous value, that variable should be set to `1.5rem`.)

Reply:
Fixed this - thanks for the catch!